### PR TITLE
Add .burger command

### PR DESCRIPTION
**Pull request by starsparrow**
Yet another addition to foods.py 

Tested in ##burgerops on snoonet

### CODE REVIEW

**Comment by linuxdaemon** [comment]
If you still want this command, I'd recommend converting it to the new `foods.py` structure

File: plugins/foods.py [hunk 1]
```diff
@@ -101,3 +453,22 @@ def func(text, action, is_nick_valid):
...
+
+    if not is_valid(user):
+        return "Sorry, I can't burg that user."
+
+    generator = textgen.TextGenerator(burger_data["templates"], burger_data["parts"], variables={"user": user})
+
+    # act out the message
+    action(generator.generate_string())    
+=======
+>>>>>>> upstream/gonzobot
```
Comment (linuxdaemon) [inline plugins/foods.py]: I believe you forgot to resolve all the conflicts during the merge

**Comment by starsparrow** [comment]
I did, sorry. I meant to get it cleaned up yesterday but was unexpectedly
busy. I'll check it out soon!

**Comment by starsparrow** [comment]
@linuxdaemon okies I think this should be good now, let me know if it needs anything else!

**Comment by starsparrow** [comment]
@linuxdaemon  also, should I be branching future silly things like this off of the python3.4 branch instead of gonzobot?

**Comment by linuxdaemon** [comment]
nah, the `python3.4` branch is the upstream CloudBot branch, the `gonzobot` branch is the one the live gonzobot instance runs off of

**Comment by starsparrow** [comment]
Ah. Should I base my branches in gonzobot-dev then or should I just keep doing it off of gonzobot?

**Comment by linuxdaemon** [comment]
Just off of the `gonzobot` branch

**Comment by edwardslabs** [comment]
This has merge conflicts that need to be fixed before it can be merged.

**Pull request merged**